flexmeasures/data/models/time_series.py

```diff
@@ -153,3 +153,7 @@ def collect(
             resolution=resolution,
             sum_multiple=sum_multiple,
         )
+
+
+class Sensor(db.Model, tb.SensorDBMixin):
+    """A sensor measures events. """
```

Comment:
Why don't we already add a `sensor_type` attribute?

Reply:
Because it's unrelated to ensuring generic sensor ids. We'll address it very soon, when we start using the Sensor table to persist a description of sensors.